pkg/kubernetes/client.go

```diff
@@ -0,0 +1,164 @@
+package kubernetes
+
+import (
+	"k8s.io/api/core/v1"
```

Comment:
go import style (at least for us, and for some go code I've seen) is put the build in imports at the top (fmt, net/http, etc), then put third-party imports second, then put our own imports third, with blank lines between the groups. gofmt will sort the group of imports for us.  But let's organize the imports this way so its quicker to read.

Reply:
+1
I let the IDE to manage the dependencies and I didn't look at it.
Will fix it.